agent_build_refactored/managed_packages/install-scriptlets/preuninstall.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Full copy of the https://github.com/scalyr/scalyr-agent-2/blob/master/installer/scripts/preuninstall.sh